src/gui/pedalboard/cables/CableView.cpp

```diff
@@ -92,16 +107,33 @@ void CableView::mouseUp (const MouseEvent& e)
     if (isDraggingCable)
     {
         cableMouse.reset();
-        connectionHelper->releaseCable (e);
+        if (connectionHelper->releaseCable (e))
+            cables.getLast()->updateEndPoint();
         isDraggingCable = false;
     }
 }
 
 void CableView::timerCallback()
 {
-    const auto mousePos = Desktop::getMousePosition() - getScreenPosition();
-    nearestPort = portLocationHelper->getNearestPort (mousePos, board);
-    repaint();
+    using namespace CableDrawingHelpers;
+
+    // repaint port glow
+    if (mouseOverClickablePort() || mouseDraggingOverOutputPort())
```

Comment:
It would be good to try to save the value of `mouseOverClickablePort()` so we don't need to call it again in the next if clause.

Reply:
Good catch, thanks